.goreleaser-fips.yml

```diff
@@ -0,0 +1,33 @@
+---
+version: 2
+project_name: nri-prometheus
+builds:
+  - id: nri-prometheus-nix-fips
+    main: ./cmd/nri-prometheus/
+    binary: nri-prometheus
+    ldflags:
+      - -s -w -X github.com/newrelic/nri-prometheus/internal/integration.Version={{.Version}} #-X main.gitCommit={{.Commit}} -X main.buildDate={{.Date}}
```

Comment:
why that piece is commented out?

Reply:
It was copy-paste. I think it can be removed. Will do. Thanks.

Reply:
done